content/v2/webhooks/events.markdown

```diff
@@ -90,6 +90,7 @@ The following events are available:
 * domain.auto\_renewal\_enable
 * domain.create
 * domain.delete
+* domain.lock
```

Comment:
Another reason for the rename. Notice how events are significantly apart which makes it even less readable.